.github/workflows/tests.yaml

```diff
@@ -1,7 +1,7 @@
 name: tests
 
 on:
-  workflow_dispatch:
+  - workflow_dispatch
```

Comment:
Let's have `on` of the same style across all workflows. It is a list here and an object in `pages`. Worth to mention, it is an object in [docs](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow).

Reply:
GitHub's official linter suggests this to be an array when the object functionality is not used

Reply:
Then please change it for pages.

> Let's have on of the same style across all workflows. It is a list here and an object in pages.

Reply:
I've standardized this across all the workflows 🎉